src/main/java/bowling/domain/frame/FinalFrame.java

```diff
@@ -0,0 +1,59 @@
+package bowling.domain.frame;
+
+import static bowling.domain.frame.NormalFrame.MAX_PITCH;
+
+public class FinalFrame extends Frame {
+
+    public static final int MAX_FINAL_PIN_COUNT = 30;
+
+    public FinalFrame(FrameRound frameRound) {
+        super(frameRound);
+    }
+
+    public static FinalFrame get() {
+        return new FinalFrame(FrameRound.last());
+    }
+
+    @Override
+    Frame next() {
+        return this;
+    }
+
+    @Override
+    public void pitch(int pinCount) {
+        validatePinCount(pinCount);
+        pins.add(Pin.from(pinCount));
+    }
+
+    private void validatePinCount(int pinCount) {
+        if ((pinCountSum() + pinCount) > MAX_FINAL_PIN_COUNT) {
+            throw new IllegalArgumentException("핀의 개수가 30개를 넘을 수 없습니다.");
+        }
+
+        if ((getPinsSize() + 1) > BONUS_PITCH) {
+            throw new IllegalArgumentException("최대 3회 투구 가능합니다.");
+        }
+    }
+
+    @Override
+    public Boolean isNextFrame() {
```

Comment:
```suggestion
    public boolean isNextFrame() {
```
참조타입을 쓰신이유가 있나요? 별다른 이유가 없으면 원시타입이 성능에서 훨씬 좋습니다. 🙂